### PR TITLE
Return raw events in stream with opt-in mapMetadata in false mode

### PR DESCRIPTION
**Pull request by fbadiola**
It allows users change the stream result by the opt-in parameter `mapMetadata` (default: true) .

mapMetadata:
- true: `onData` callback receive an object that its keys will be the column 
- false: `onData` callback receive the raw array that API V2 send.

### CODE REVIEW

File: lib/oboeRequest.js [hunk 1]
```diff
@@ -23,13 +23,14 @@ class OboeRequest {
    * Make a streaming query to the Devo API.
    *
    * @param {Object} options An object with method, body and headers.
+   * @param {Object} processParams An object with stream process opt-in parameters
```
Comment (alexfernandez) [inline lib/oboeRequest.js]: Isn't it better to pass the options to the constructor `OboeRequest()` and store as a private attribute `_parseMetadata`?

Comment (fbadiola) [reply lib/oboeRequest.js]: Yes, it's done.

Comment (fbadiola) [reply lib/oboeRequest.js]: I closed the js-helper PR.

Comment (alexfernandez) [reply lib/oboeRequest.js]: Please [document the new option](https://github.com/DevoInc/browser-sdk#query-options), and update version accordingly. Thanks! :)

Comment (fbadiola) [reply lib/oboeRequest.js]: Ready. Checkout please.

Comment (alexfernandez) [reply lib/oboeRequest.js]: You still need to update the version.

Comment (fbadiola) [reply lib/oboeRequest.js]: Ready, changed to 1.2.6

**Pull request merged**